projects/nutanix-cloud-native/cluster-api-provider-nutanix/README.md

```diff
@@ -0,0 +1,17 @@
+## **Cluster API Provider for Nutanix**
+![Version](https://img.shields.io/badge/version-v0.5.0-blue)
+
+The [Cluster API Provider for Nutanix (CAPX)](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix) is a the implementation of Cluster API for Nutanix.
+
+
+### Updating
+
+1. Review releases and changelogs in upstream [repo](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix) and decide on the new version.
+   Please review carefully and if there are questions about changes necessary to eks-anywhere to support the new version
+   and/or automatically update between eks-anywhere version reach out to @jaxesn, @pokearu or @abhnvp.
+2. Update the `GIT_TAG` file to have the new desired version based on the upstream release tags.
+3. Check the go.mod file to see if the golang version has changed when updating a version. Update the `GOLANG_VERSION` in `Makefile` to match the version upstream.
```

Comment:
I dont like pointing to the go.mod as the source of truth for golang versions. Is there a better file in the CAPX repo, maybe the release job or makefile or dockerfile?

Reply:
Not really; can you elaborate on why we don't like go.mod specifying the version of Go?

Reply:
That is because, the `go` directive in a go.mod file represents the minimum Go version that's required to compile that source code. For example, if your code uses generics, both your Go mod version and actual Go compile toolchain must be 1.18 because generics was added only in 1.18. But if there's nothing specific to 1.18 in the code, the go mod may contain 1.17 and still be compiled with 1.18 or anything above it. The ideal version for build-tooling should be, as Jackson stated above, the one you use in Makefile/Dockerfile/goreleaser.yml for go build